.github/workflows/graphql_build.yml

```diff
@@ -21,5 +21,5 @@ jobs:
       - name: Code analyze check
         run: |
           cd packages/graphql
-          dart analyze lib
-          dart analyze test
\ No newline at end of file
+          dart analyze lib --fatal-infos
+          dart analyze test --fatal-infos
```

Comment:
Can you add some context on why we're doing this?

Reply:
for some reason, we are missing dart code error and the test doesn't fail, this increases the level of failure, do you think that it is useful?